doc/modules/model_evaluation.rst

```diff
@@ -1918,25 +1918,25 @@ the actual formulas).
 .. dropdown:: Mathematical divergences
 
   The positive likelihood ratio is undefined when :math:`fp = 0`, which can be
-  interpreted as the classifier perfectly identifying positive cases. If :math:`fp
-  = 0` and additionally :math:`tp = 0`, this leads to a zero/zero division. This
-  happens, for instance, when using a `DummyClassifier` that always predicts the
+  interpreted as the classifier never wrongly identifying negative cases as positives.
+  This happens, for instance, when using a `DummyClassifier` that always predicts the
```

Comment:
Maybe my point would be clearer if we give some examples:
- Case 1: fp=0, tp≠0, LR_+ diverges
  - the classifier is perfect
  - in coherence with being a "higher is better" metric e.g.:
  - y_true = np.array([0, 0, 0, 1, 1, 1])
  - y_pred = np.array([0, 0, 0, 1, 1, 1])
 - Case 2: fp=0, tp=0, LR_+ diverges
   - y_pred only includes the majority class
   - can be misleading as it's the case of a `DummyClassifier` with imbalanced data e.g.:
   - y_true = np.array([0, 0, 0, 0, 0, 1])
   - y_pred = np.array([0, 0, 0, 0, 0, 0])
 - Case 3: fn=0, tp=0, LR_+ and LR_- diverge
   - the test set only includes the majority class, but the classifier still makes errors
   - can be misleading as it's a common case when cross-validating with imbalanced data e.g.:
   - y_true = np.array([0, 0, 0, 0, 0, 0])
   - y_pred = np.array([0, 0, 0, 0, 0, 1])
 - Case 4: tn=0, LR_- diverges
   - positive and negative classes are inverted
   - can be indicative of divergences in LR_+ if classes are reverted e.g.:
   - y_true = np.array([1, 1, 1, 0, 0, 0])
   - y_pred = np.array([0, 0, 0, 1, 1, 1])
 
Having this in the shape of table would be nice.

Reply:
Something similar to:
```
| Zeros in the Confusion Matrix | Diverges       | Reason                                                         | Interpretation                                                                                     | Example                                      |
|-------------------------------|----------------|----------------------------------------------------------------|----------------------------------------------------------------------------------------------------|----------------------------------------------|
| fp=0, tp≠0                    | LR_+          | The classifier is perfect                                      | In coherence with being a "higher is better" metric                                               | `y_true = [0, 0, 0, 1, 1, 1]`<br>`y_pred = [0, 0, 0, 1, 1, 1]` |
| fp=0, tp=0                    | LR_+          | `y_pred` only includes the majority class                      | Can be misleading, as it’s the case of a `DummyClassifier` with imbalanced data                   | `y_true = [0, 0, 0, 0, 0, 1]`<br>`y_pred = [0, 0, 0, 0, 0, 0]` |
| fn=0, tp=0                    | LR_+ and LR_- | The test set only includes the majority class, but classifier still makes errors | Can be misleading, common when cross-validating with imbalanced data | `y_true = [0, 0, 0, 0, 0, 0]`<br>`y_pred = [0, 0, 0, 0, 0, 1]` |
| tn=0                          | LR_-          | Positive and negative classes are inverted                     | Can indicate divergences in LR_+ if classes are reversed                                          | `y_true = [1, 1, 1, 0, 0, 0]`<br>`y_pred = [0, 0, 0, 1, 1, 1]` |
```

Reply:
Thank you for clarifying this, @ArturoAmorQ. I think I understand for the `LR+` case (not so sure about the `LR-` case though, so I didn't touch it).

I have tried to point out these two cases `fp==0` might indicate and also stay in the scope of a side note/drop down on mathematical divergences. I feel that this could be material for an extensive example, but this is outside the scope of this PR. 

Please let me know what you think about my attempt to rephrase and give advice on how to interpret the positive likelihood ratio in case of a divergence / warning shown to the user.